app/src/main/res/menu/feedlist.xml

```diff
@@ -22,6 +22,14 @@
         android:title="@string/load_complete_feed"
         custom:showAsAction="collapseActionView">
     </item>
+
+    <item
+        android:id="@+id/action_search"
+        android:icon="?attr/action_search"
+        custom:showAsAction="collapseActionView|ifRoom"
```

Comment:
Unless we need the room, can we make this 'always' instead of 'ifRoom', it will preserve the existing behavior.  I think the queue is the only spot that needs 'ifRoom'.